dependencies.json

```diff
@@ -0,0 +1,11 @@
+{
+    "recipesEndpoint": "",
+    "packages": [
+        {
+            "requirement": "dev-ibx-8811-SA-sessions-sharing as 5.0.x-dev",
+            "repositoryUrl": "https://github.com/ibexa/core",
+            "package": "ibexa/core",
+            "shouldBeAddedAsVCS": true
```

Comment:
AFAIR, value for this parameter should be set to false because `ibexa/core` has public access. This should fix red CI.